### ISSUE TITLE
Category visual problem 

### ISSUE THREAD

**Issue by Verconas** [tachiyomiorg/tachiyomi#7043]
### Steps to reproduce

When i open the app and i go to a category the bar which indicates the categry slides far to the left and it stays like that if i change the category it indicates the new one but goes back to the left again 

### Expected behavior

When i'm using a certain category the bar on top of it should stay fixed on that one category 

### Actual behavior

Instead the categories on top goes to the left most category

### Tachiyomi version

0.13.4

### Android version

Android 10

### Device

Huawei y9 prime 

### Other details

Instead of this
![Screenshot_20220429_142722](https://user-images.githubusercontent.com/104640486/165976574-20809f9a-adf7-4b6f-a145-5cbf69f7c077.jpg)
This happens 
![Screenshot_20220429_142739](https://user-images.githubusercontent.com/104640486/165976588-3bdceef4-7e8e-46e7-b90e-d9421ffdf6f0.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.13.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by FourTOne5** [comment]
Simple step to reproduce:
1. Choose a category which is on the right side
2. Close the app, remove from recent and start app again

Other info: 
Swiping to another category fixes the placement of categories on top

**Comment by tanuukii** [comment]
Other step to reproduce:
1. Navigate to the rightmost category
2. Click Browse icon and go back to Library

I think this was introduced after #6591 was merged

**Comment by mm12** [comment]
I wonder if this will fix #6860
